inky/inky.go

```diff
@@ -106,6 +113,44 @@ type Opts struct {
 	BorderColor Color
 }
 
+// DetectOpts tries to read the device opts from EEPROM, it is recommended to use I2C bus '1'.
```

Comment:
> it is recommended to use I2C bus '1'.

I'd recommend to remove.

Reply:
That's a fair point, I was just pulling it from the Periph Raspberry Pi setup recommendations, but I can see how that should probably be kept out of function documentation